### DISCUSSION TITLE
IP Location Check

### DISCUSSION THREAD

**Discussion by UPSOKen** [VueTorrent/VueTorrent#801] [Ideas]
Not sure if this is possible, but it would be really great if there was a way to display a "IP + IP Location" in the statistics area on the left (Maybe a new section underneath Session Stats). Would be great to be able to check if proxies or VPNs are configured and working correctly.

Thanks for all the work everyone has done. Love this project!

**Comment by WDaan** [comment]
Like #532?

**Reply by UPSOKen** [reply]
Yes Sir. Don't know how I missed that 🤦‍♂️

The only thing I would add is an IP location database check to possibly display for quickly checking VPN connections.
